Validation/RecoB/plugins/BDHadronTrackMonitoringHarvester.cc

```diff
@@ -0,0 +1,212 @@
+#include "Validation/RecoB/plugins/BDHadronTrackMonitoringHarvester.h"
+
+// intialize category map
+//std::map<unsigned int, std::string> TrkHistCat(map_start_values, map_start_values + map_start_values_size);
```

Comment:
@davidlange6 
Here I am not sure, when I re-initialize the map in the Harverster.cc, the compiler complains about a redefinition:
 multiple definition of `TrkHistCat'. I think this is because I include the Analyzer.h in the Harvester.h file, but it already has a definition of TrkHistCat in the Analyzer.cc.

Do you think the Harvester knows to take the definition in Analyzer.cc? When I do the test the output looks fine...

Cheers
Seth